UnitedSets/Windows/MainWindow.xaml.EventHandler.cs

```diff
@@ -61,13 +51,42 @@ void FirstRun()
         icon.DangerousAddRef(ref success);
         PInvoke.SendMessage(WindowEx.Handle, PInvoke.WM_SETICON, 1, icon.DangerousGetHandle());
         PInvoke.SendMessage(WindowEx.Handle, PInvoke.WM_SETICON, 0, icon.DangerousGetHandle());
+		}
 
         if (Keyboard.IsShiftDown)
             WindowEx.SetAppId($"UnitedSets {WindowEx.Handle}");
 		Cell.ValidDrop += Cell_ValidDrop;
-		if (FeatureFlags.USE_TRANSPARENT_WINDOW)
-			TransparentFinalize();
+		HandleCLICmds();
+	}
+	async void HandleCLICmds() {
+		var toAdd = CLI.GetArrVal("add-window-by-exe");
+		var editLastAddedWindow = CLI.GetFlag("edit-last-added");
+		LeftFlyout.NoAutoClose = CLI.GetFlag("edit-no-autoclose");
+		var profile = CLI.GetVal("profile");
+		if (!String.IsNullOrWhiteSpace(profile)) {
+			if (Path.HasExtension(profile) == false)
+				profile += ".json";
+			if (! File.Exists(profile) && ! Path.IsPathRooted(profile) )
+				profile = Path.Combine(USConfig.BaseProfileFolder,profile);
+			if (File.Exists(profile)) {
+				await Task.Delay(1500);
+				await persistantService.ImportSettings(profile);
+			}
+
 
+		}
+
+		foreach (var itm in toAdd) {
+			var procs = System.Diagnostics.Process.GetProcesses().Where(p => p.ProcessName.Equals(itm, StringComparison.OrdinalIgnoreCase)).ToList();
+			foreach (var proc in procs) {
+				if (!proc.HasExited)
+					AddTab( WindowEx.FromWindowHandle(proc.MainWindowHandle));
+			}
+		}
+		if (editLastAddedWindow && Tabs.Count > 0)
+			Tabs.Last().TabDoubleTapped(this, new Microsoft.UI.Xaml.Input.DoubleTappedRoutedEventArgs());
+		
+	}
 	}
```

Comment:
Double `}` at the end, causing compile error

Reply:
Sorry I did selective staging to make the commits a bit cleaner, I caught this pretty fast but I think you had already cloned it by then.